src/js/contexts/MessageContext/MessageContext.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import defaultMessages from '../../languages/default.json';
+
+// opts:
```

Comment:
```suggestion
// prop options:
```
I think?!

Reply:
It's not a prop. Just an options argument to the function